worker/deploy/README.md

```diff
@@ -1,26 +1,31 @@
 # Worker Node deployment
 
-Arrebol tasks are processed by Workers. In a typical deployment, a few Workers are deployed together in a single virtual machine, a Worker Node. As each task run in an isolated docker container, the Docker engine and other dependencies must have be installed and configured in the Worker Node. Below, we described how to configure the Worker Node.
+Arrebol tasks are processed by Workers. In a typical deployment, a few Workers are deployed together in a single virtual machine, a `Worker Node`. As each task run in an isolated docker container, the Docker engine and other dependencies must have be installed and configured in the Worker Node. Below, we described how to configure the Worker Node.
 
 ## Requeriments
 
-Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible automation tool](https://www.ansible.com/) should be installed in the deploy coordination host.
+Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible](https://www.ansible.com/) automation tool should be installed in the deploy `coordination host`.
+To install Ansible run these commands:
+```bash
+sudo apt update
+sudo apt install software-properties-common
+sudo apt-add-repository --yes --update ppa:ansible/ansible
+sudo apt install -y ansible
+```
 
-`Warning:` Deployment does not work for nodes with Ubuntu version 14 or earlier.
+`Note: Deployment does not work for nodes with Ubuntu version 14 or earlier.`
```

Comment:
its ambiguous, you could rewrite it like: The deployment requires versions newer than 14 for ubuntu.

Reply:
done.